daml-lf/language/src/main/scala/com/digitalasset/daml/lf/language/EvaluationOrder.scala

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.lf.language
```

Comment:
```suggestion
package com.daml.lf.language
```
maybe we could put it in some "features" or "devfeature" package

Reply:
Hmm actually that file name is a mistake: I intended to call it LanguageDevConfig.scala. Done.

Would you still prefer to move it to its own package?

Reply:
I would prefer those definitions mention `dev` in their full name in order to make explicit they are only relevant for Dev.
Adding Dev in their name, put it inside a special package or  inside a object `LanguageDevConfig` would be enough.  
I have to choose I will go for the later. 

This is a minor remark, I let you judge.

Reply:
Done